### PR TITLE
📈 Add signal resources

### PR DESCRIPTION
**Pull request by bertfroeba**
Request to add a couple of simple signal resources that are a bit more dynamic and interesting.

Triangle Wave
Screenshot 2019-12-17 at 11 00 11

Saw Wave
Screenshot 2019-12-17 at 10 37 49

Also, using content type allows to be more descriptive in the Portal:
Screenshot 2019-12-17 at 11 00 32

### CODE REVIEW

**Comment by iot-pdmc** [comment]
Thanks for your changes.
@ARMmbed/pdmc-integrators please review.

File: main.cpp [hunk 1]
```diff
@@ -214,7 +241,7 @@ int main(void)
         } else if (in_char == 'r') {
             (void) fcc_storage_delete(); // When 'r' is pressed, erase storage and reboot the board.
             printf("Storage erased, rebooting the device.\n\n");
-            wait(1);
+            thread_sleep_for(1000);
```
Comment (JanneKiiskila) [inline main.cpp]: Is there some special reason for changing the `wait` out for the `thread_sleep_for`?

Comment (bertfroeba) [reply main.cpp]: i saw a deprecation notice in mbed studio for wait
```
MBED_DEPRECATED_SINCE("mbed-os-5.14",
                      "'wait' is deprecated in favor of explicit sleep functions. To sleep, 'wait' should be replaced by "
                      "'ThisThread::sleep_for' (C++) or 'thread_sleep_for' (C). If you wish to wait (without sleeping), call "
                      "'wait_us'. 'wait_us' is safe to call from ISR context.")
```

Comment (JanneKiiskila) [reply main.cpp]: That is definitely a good reason!-)

**Comment by MarceloSalazar** [comment]
I like the idea of generating a more interesting waveform, but really don't want us to increase the length of the app nor another resource.
My proposal is to re-use the existing resource `3202/0/5600` so we can refer to this in the pelion-quick-start - point 4.3:
https://os.mbed.com/guides/connect-device-to-pelion/4/?board=ST-Discovery-L475E-IOT01A

**Comment by loverdeg-ep** [comment]
On the topic of adding objects.

What about the Event Log or one of the connectivity objects?
Those should be a near default part of the client anyway and would be really nice touches.

**Pull request closed without merging**